hedera-node/hedera-app-spi/src/main/java/com/hedera/node/app/spi/validation/ExpiryMeta.java

```diff
@@ -68,8 +94,8 @@ public boolean hasAutoRenewPeriod() {
      *
      * @return whether this metadata has explicit auto-renew account
      */
-    public boolean hasAutoRenewNum() {
-        return autoRenewNum != NA;
+    public boolean hasAutoRenewAccountId() {
+        return autoRenewAccountId.accountNum() != NA;
```

Comment:
autoRenewAccountId != null

Reply:
There are many places using new ExpiryMeta(
                op.hasExpirationTime() ? op.expirationTime().seconds() : NA,
                op.hasAutoRenewPeriod() ? op.autoRenewPeriod().seconds() : NA,
                NA); 
If I change it to check null here, all those places need to be updated. Do you think null is better here?

Reply:
Updated.